myapp/spec/system/tasks_spec.rb

```diff
@@ -1,113 +1,302 @@
-      expect(Task.count).to eq 0
+RSpec.describe 'Test cases for Task :', type: :system do
+  describe 'In task list page' do
+    context 'when there is no task,' do
+      before do
+        # 一覧画面を開く
+        visit tasks_path
+      end
+      it 'does not show any task' do
+        # 正しい情報が表示されていること
+        expect(page).to have_content 'タスク一覧'
+        expect(Task.count).to eq 0
+      end
     end
 
-    it 'shows correct result when there are multiple tasks' do
-      @task1 = Task.create!(title: 'Spec', description: 'test')
-      @task2 = Task.create!(title: 'Spec2', description: 'test2')
+    context 'when there are tasks,' do
+      let!(:task1) { FactoryBot.create(:task) }
+      let!(:task2) { FactoryBot.create(:task, title: 'Spec2', description: 'test2') }
+
+      before do
+        # 一覧画面を開く
+        visit tasks_path
+      end
+      it 'shows correct result when there are multiple tasks' do
+        # 正しい情報が表示されていること
+        expect(page).to have_content 'タスク一覧'
+        expect(page).to have_content 'Spec'
+        expect(page).to have_content 'test'
+        expect(page).to have_content 'Spec2'
+        expect(page).to have_content 'test2'
+        expect(Task.count).to eq 2
+      end
+    end
 
-      # 一覧画面を開く
-      visit tasks_path
+    context 'when error,' do
+      before do
+        allow(Task).to receive(:all).and_raise(RuntimeError)
+        # 一覧画面を開く
+        visit tasks_path
+      end
 
-      # 正しい情報が表示されていること
-      expect(page).to have_content 'Tasks'
-      expect(page).to have_content 'Spec'
-      expect(page).to have_content 'test'
-      expect(page).to have_content 'Spec2'
-      expect(page).to have_content 'test2'
-      expect(Task.count).to eq 2
+      it 'redirected to 500 error page' do
+        expect(page).to have_content '伍〇〇'
+      end
     end
   end
 
-  describe 'New task page' do
-    it 'create task correctly' do
-      # 新規画面を開く
-      visit new_task_path
+  describe 'In new task page' do
+    context 'when success,' do
+      it 'create task correctly' do
+        # 新規画面を開く
+        visit new_task_path
+
+        # 新規画面が開いてること（titleが空白になってる）
+        expect(page).to have_content '新規登録'
+        expect(find_field('task_title').text).to be_blank
+        expect(page).to have_field 'task_description', with: ''
+
+        # titleとdescriptionを入力
+        fill_in 'task_title', with: 'Spec test new task'
+        fill_in 'task_description', with: 'Spec test new task description'
+
+        # 登録
+        click_button 'タスクを登録する'
+
+        # 正しく登録されていること
+        expect(page).to have_content 'タスクを作成しました'
+        expect(page).to have_content 'タスク詳細'
+        expect(page).to have_content 'Spec test new task'
+        expect(page).to have_content 'Spec test new task description'
+      end
+    end
+    context 'when fail,' do
+      before do
+        allow_any_instance_of(Task).to receive(:save).and_return(false)
+      end
+      it 'shows error message' do
```

Comment:
A new line is needed under `before` block!

```suggestion
      end

      it 'shows error message' do
```

Reply:
The same is true for other parts of the this file.
You could use `rubocop` to fix it easily.

Reply:
Idk why but `rubocop -A` didn't fix this for me.

I fixed this and checked other parts.

Reply:
Thank you!

> Idk why but rubocop -A didn't fix this for me.

Oh, really? I wonder why.